services/src/main/java/org/exoplatform/webconferencing/listeners/CallRecordingNotificationListener.java

```diff
@@ -42,19 +42,23 @@ public void onEvent(Event<CallInfo, Map<? extends String, ? extends String>> eve
         Map<? extends String, ? extends String> info = event.getData();
         List<String> participants = callInfo.getParticipants().stream().map(UserInfo::getId).collect(Collectors.toList());
         String status = info.get("status");
-        String uploadingUser = info.get("user_id");
         String fileName = info.get("file_name") == null ? "" : info.get("file_name");
         String identity =  info.get("identity") == null ? null : info.get("identity");
         String callType = info.get("type") == null ? "" : info.get("type");
         WebConferencingService webConferencingService = ExoContainerContext.getCurrentContainer().getComponentInstanceOfType(WebConferencingService.class);
+        Map<String, String> identityInfo = webConferencingService.getNameAndAvatarByIdentityAndType(callType, identity);
+        String name = identityInfo.get("name") == null ? "" : identityInfo.get("name");
+        String avatarUrl = identityInfo.get("avatar_url") == null ? "" : identityInfo.get("avatar_url");
         if(callType.equals(WebConferencingService.OWNER_TYPE_SPACE) || callType.equals(WebConferencingService.OWNER_TYPE_SPACEEVENT)) {
             NotificationContext ctx = NotificationContextImpl.cloneInstance();
-            String fileUrl = webConferencingService.getRecordingUrl(uploadingUser, fileName, callType, identity);
+            String fileUrl = webConferencingService.getRecordingUrl(null, fileName, callType, identity);
```

Comment:
why do you remove the first param ?

Reply:
> why do you remove the first param ?

car in case of space call or event call we no need to use this parameter uploadingUser to get recorded file URL .

Reply:
should we provide the uploadinUser parameter in all cases, and use it only when necessary ?